pkg/charts/package.go

```diff
@@ -21,10 +19,8 @@ type Package struct {
 	Name string `yaml:"name"`
 	// PackageVersion represents the current version of the package. It needs to be incremented whenever there are changes
 	PackageVersion int `yaml:"packageVersion"`
-	// ReleaseCandidateVersion represents the version of the release candidate for a given package.
-	ReleaseCandidateVersion int `yaml:"releaseCandidateVersion"`
 	// AdditionalCharts are other charts that should be packaged together with this
-	AdditionalCharts []AdditionalChart `yaml:"additionalCharts,omitempty"`
+	AdditionalCharts []*AdditionalChart `yaml:"additionalCharts,omitempty"`
```

Comment:
Required to ensure upstreamChartVersion is preserved, since pass-by-value would not preserve this.